include/MySQL_Thread.h

```diff
@@ -335,6 +335,7 @@ class MySQL_Threads_Handler
 	MySQL_Listeners_Manager *MLM;
 	public:
 	struct {
+		bool azure_enabled;
```

Comment:
Please check the Qianqian if this name follows the public conversion as other open source tools.

Reply:
The connection attribute field is defined by MySQL protocol, I think the MySQL drivers can set the key-value pairs for this field as long as it does not exceed the length limitation